08-transport.md

```diff
@@ -248,7 +248,7 @@ and `16 bytes` for the `poly1305` tag.
        handshake digest.
 
 
-  * Send `m = 0 || e || c` to the responder over the network buffer.
+  * Send `m = 0 || e.pub.serializeCompressed().pub || c` to the responder over the network buffer.
```

Comment:
Hmm, thought I described the behavior/return-value of `.serializeCompressed()`, but looking at it again it only declares the function rather than specifying it's behavior.